Containerfile.plugin

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Red Hat
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# --
+
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_golang_1.22  AS builder
+
+ENV LANG=en_US.utf8
+ENV GIT_COMMITTER_NAME devtools
+ENV GIT_COMMITTER_EMAIL devtools@redhat.com
+LABEL com.redhat.delivery.appregistry=true
+
+WORKDIR /gitops-operator
+
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY cmd/main.go cmd/main.go
+COPY api/ api/
+COPY controllers/ controllers/
+COPY common/ common/
+COPY version/ version/
+
+ARG VERBOSE=2
+ENV GOFLAGS="-mod=mod"
+RUN go build -o bin/manager ./cmd/main.go
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal
+ENV LANG=en_US.utf8
+COPY --from=builder /gitops-operator/bin/manager /usr/local/bin/manager
+# Redis HA script templates
+COPY build/redis /var/lib/redis
+USER 10001
+ENTRYPOINT [ "/usr/local/bin/manager" ]
+
+LABEL \
+    name="openshift-gitops-1/gitops-rhel8-operator" \
+    version=${CI_CONTAINER_VERSION} \
+    License="Apache 2.0" \
+    com.redhat.component="openshift-gitops-operator-container" \
+    com.redhat.delivery.appregistry="false" \
+    release=${CI_CONTAINER_RELEASE} \
+    upstream-version=${CI_UPSTREAM_VERSION} \
+    upstream-vcs-ref="${CI_GITOPS_OPERATOR_UPSTREAM_COMMIT}" \
```

Comment:
Same as https://github.com/redhat-developer/gitops-backend/pull/47#discussion_r1850019207
> I think we need to find replacements for `CI_*` env vars. If I am not wrong, these vars are CPAAS specific. We need to figure out konflux equivalent for these or manually define them.

Reply:
We can address this during release pipeline configurations, shouldn't be a blocker for merging this PR.